### PR TITLE
Add test for model variants

### PR DESCRIPTION
**Pull request by ChHaeni**

### CODE REVIEW

File: t/run-complex-model-with-variants.t [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+use Agrammon::OutputFormatter::GUI;
+use Agrammon::OutputFormatter::Text;
+use Agrammon::TechnicalParser;
+
+my $temp-dir = $*TMPDIR.add('agrammon_testing');
+
+#| Model version and variants
+my $model-version = 'hr-inclNOxExtendedWithFilters';
+my $model-variants = <'Single-default' 'Kantonal-LU'>;
+# my $model-variants = <Single>;
```
Comment (zaucker) [inline t/run-complex-model-with-variants.t:17]:
```
my @model-variants = ( 'Single-default', 'Kantonal-LU');
...
for @model-variants -> $variant {
}
```
might be slightly simpler.

File: t/run-complex-model-with-variants.t [hunk 2]
```diff
@@ -0,0 +1,82 @@
+use Test;
+use Data::Dump::Tree;
+use Agrammon::DataSource::CSV;
+use Agrammon::Model;
+use Agrammon::ModelCache;
+use Agrammon::Model::Parameters;
+use Agrammon::OutputFormatter::CSV;
+use Agrammon::OutputFormatter::GUI;
```
Comment (zaucker) [inline t/run-complex-model-with-variants.t:8]: Please use `Agrammon::OutputFormatter::JSON` changed recently, see current master.

**Review by zaucker — CHANGES_REQUESTED** [review]

File: t/run-complex-model-with-variants.t (hunk 1 above)
Comment (ChHaeni) [reply t/run-complex-model-with-variants.t]: included

File: t/run-complex-model-with-variants.t (hunk 2 above)
Comment (ChHaeni) [reply t/run-complex-model-with-variants.t]: included

**Review by zaucker — APPROVED** [review]

**Pull request merged**